packages/docs/components/tab.md

```diff
@@ -25,16 +22,309 @@ links:
+        </blockquote>
+      </template>
+    </OdsTabs>
+  </template>
+</Visual>
+
+## Usage
+
+<Description>
+
+Tabs were created to shorten long pages. Before you use these, we’d recommend laying 
+out all of the content on your page out first. From there figure out common themes and 
+see what could be grouped together. Those themes should become your Tabs.
+
+</Description>
+
+### Relate to the page title
+
+<Description>
+
+**Tabs are not navigation**. Meaning they don’t take you from place to place. Rather, they are meant for context switching related to the page.
+
+</Description>
+
+<Visual variant="positive" content="no-end" class="is-tab-small-sample">
+  <header>
+    <h2>Terrestrial Planets</h2>
+    <p>Terrestrial planets are planets that are composed primarily of silicate rocks or metals.</p>
+  </header>
+  <OdsTabs aria-label="Types of terrestrial planets" :active="tabsPlanets.active" :tablist="tabsPlanets.tablist" id="example-2"></OdsTabs>
+</Visual>
+
+<Visual variant="negative" content="no-end" class="is-tab-small-sample">
+  <header>
+    <h2>Terrestrial Planets</h2>
+    <p>Terrestrial planets are planets that are composed primarily of silicate rocks or metals.</p>
+  </header>
+  <OdsTabs aria-label="Famous constellations" :active="tabsConstellations.active" :tablist="tabsConstellations.tablist" id="example-3"></OdsTabs>
+</Visual>
+
+### Position above major content
+
+<Description>
+
+Tabs are best used at the top of the page or situated above the content it’s related to. This will help establish hierarchy.
+
+</Description>
+
+
+<Visual variant="positive" content="no-end" class="is-tab-small-sample">
+  <header>
+    <h2>Missions</h2>
+    <p>There have been 49 missions involving various types of spacecraft.</p>
+  </header>
+  <OdsTabs aria-label="Missions by type" :active="tabs.active" :tablist="tabs.tablist" id="example-4"></OdsTabs>
+</Visual>
+
+<Visual variant="negative"  content="no-end" class="is-tab-small-sample">
+  <OdsTabs aria-label="Missions by type" :active="tabs.active" :tablist="tabs.tablist" id="example-5">
+  <template slot="tab-orbiter">
+    <header>
+      <h2>Missions</h2>
+      <p>There have been 8 missions involving orbiters.</p>
+    </header>
+  </template>
+  <template slot="tab-atmospheric">
+    <header>
+      <h2>Missions</h2>
+      <p>There have been 12 missions involving atmospheric vehicles.</p>
+    </header>
+  </template>
+  <template slot="tab-lander">
+    <header>
+      <h2>Missions</h2>
+      <p>There have been 4 missions involving lander vehicles.</p>
+    </header>
+  </template>
+  </OdsTabs>
+</Visual>
+
+### Don't stack tabs
+
+<Description>
+
+Use tabs sparingly. Limit Tabs to one per page, and refrain from including a second set of Tabs within a tabpanel.
```

Comment:
Tabs or tabs? Need to stick to one convention.

Reply:
Good call, fixed.